### PR TITLE
[New Rule] Microsoft IIS Service Account Password Dumped

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/166

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+risk_score = 73
+rule_id = "0564fb9d-90b9-4234-a411-82a546dc1343"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+  (process.name:appcmd.exe  or process.pe.original_file_name:appcmd.exe) and
+  process.args:(/[lL][iI][sS][tT]/ and //[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
```
Comment (brokensound77) [inline rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml:29]:
```suggestion
event.category:process AND event.type:(start OR process_started) AND
  (process.name:appcmd.exe  OR process.pe.original_file_name:appcmd.exe) AND
  process.args:(/[lL][iI][sS][tT]/ AND //[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
```

lucene requires uppercase

File: rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+risk_score = 73
+rule_id = "0564fb9d-90b9-4234-a411-82a546dc1343"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process AND event.type:(start OR process_started) AND
+  (process.name:appcmd.exe  OR process.pe.original_file_name:appcmd.exe) AND
+  process.args:(/[lL][iI][sS][tT]/ AND //[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
```
Comment (brokensound77) [inline rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml:29]:
```suggestion
  process.args:(/[lL][iI][sS][tT]/ AND /\/[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
```

can you test this to make sure it is correct

File: rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with Microsoft IIS webserver access via a web shell or alike can decrypt and dump IIS Appool service account
+password using appcmd command. Verify context of this execution such as parent process, file creation, username to
+validate the legitimacy of this action.
```
Comment (bm11100) [inline rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml:12]:
```suggestion
Identifies the Internet Information Services (IIS) command-line tool, AppCmd, being used to list passwords. An attacker with IIS web server access via a web shell can decrypt and dump the IIS AppPool service account
password using AppCmd. 
```
The below could be moved to `note` - 
> It's recommended to verify the context of this execution, such as parent process name, file creations, and/or username to validate the legitimacy of this action.

Comment (bm11100) [reply rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml:12]: probably need to lint if you take this recommendation

File: rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+references = ["https://blog.netspi.com/decrypting-iis-passwords-to-break-out-of-the-dmz-part-1/"]
+risk_score = 73
+rule_id = "0564fb9d-90b9-4234-a411-82a546dc1343"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process AND event.type:(start OR process_started) AND
+  (process.name:appcmd.exe  OR process.pe.original_file_name:appcmd.exe) AND
```
Comment (threat-punter) [inline rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml:27]:
```suggestion
  (process.name:appcmd.exe OR process.pe.original_file_name:appcmd.exe) AND
```

**Review by threat-punter — APPROVED** [review]
🤗
Just needs linting before it's merged

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**